scripts/mkfiles.sh

```diff
@@ -36,6 +36,12 @@ case $OS in
   * )            DATE="date";;
 esac
 
+SED=
+case $OS in
+  Darwin )	 SED="gsed";;
```

Comment:
This was actually the most important change to get this to work on Mac.  Apparently, BSD sed is odd.  In order to get this set up, I had to do "brew install gnu-sed".  Also, for gdate, I had to do "brew install coreutils".  I figure if you want to add this, I could also update the setup documentation to include something like "on a Mac, run these two brew commands to get the gnu tools you need..."

Reply:
Nice one!  It's funny that the original IOHK scripts had some logic around DARWIN for `date`, but not for `sed`

Reply:
Yeah, I thought that was an odd oversight.  It blew up rather immediately, so I am guessing they never ran it on a mac.